File: python/Sequencial/EX8_A.py
#8. Faça um programa que receba o valor de um depósito
# e o valor da taxa de juros, calcule e mostre o
# valor do rendimento e o valor total depois do rendimento.

print("***********************SEQUENCIAL*****************************")
print("************************Programa*8****************************")
print("*******Calcula deposito com taxa de juros e rendimento********")

vlr_dep = float(input("Digite o valor do deposito: "))
vlr_tx  = int(input("Digite o valor do imposto. Ex 10 que é igual a 10%: "))

vlr_rendimento = vlr_dep * vlr_tx/100
vlr_total = vlr_dep + vlr_rendimento

print("O valor do rendimento é: R$ {: .2f} | O valor Total é: R$ {: .2f}".format(float(vlr_rendimento), float(vlr_total)))

File: JS/Sequencial/EX8_A.js
/*
8. Faça um programa que receba o valor de um depósito e o valor da taxa de juros, calcule e mostre o
valor do rendimento e o valor total depois do rendimento.
*/

var vlr_deposito = prompt("Digite o valor do depóstio")
var vlr_tx = prompt("Digite o valor da taxa de juros. Ex 10 é igual a 10%")

vlr_deposito = parseFloat(vlr_deposito)
vlr_tx = parseInt(vlr_tx)

var vlr_rendimento = vlr_deposito * vlr_tx/100
var vlr_total = vlr_rendimento + vlr_deposito

vlr_rendimento = parseFloat(vlr_rendimento)


console.log("O valor do rendimento é "+ vlr_rendimento.toFixed(2)+" | "+"O valor total do depósito é: "+ vlr_total.toFixed(2))